go/skynet.go

```diff
@@ -4,27 +4,28 @@ import "fmt"
 import "time"
 
 func skynet(c chan int, num int, size int, div int) {
-    if (size == 1) {
-        c <- num
-    } else {
-        rc := make(chan int)
-        sum := 0
-        for i := 0; i < div; i++ {
-            sub_num := num + i * (size / div)
-            go skynet(rc, sub_num, size / div, div)
-        }
-        for i := 0; i < div; i++ {
-            sum += <-rc
-        }
-        c <- sum
-    }
+	if size == 1 {
+		c <- num
+		return
+	}
+
+	rc := make(chan int)
+	var sum int
+	for i := 0; i < div; i++ {
+		subNum := num + i*(size/div)
+		go skynet(rc, subNum, size/div, div)
+	}
+	for i := 0; i < div; i++ {
+		sum += <-rc
+	}
+	c <- sum
 }
 
 func main() {
-    c := make(chan int)
-    start := time.Now().UnixNano() / 1000000 
-    go skynet(c, 0, 1000000, 10)
-    result := <-c
-    end := time.Now().UnixNano() / 1000000
-    fmt.Printf("Result: %d in %d ms.\n", result, end - start)
+	c := make(chan int)
+	start := time.Now()
+	go skynet(c, 0, 1000000, 10)
+	result := <-c
+	took := time.Since(start)
+	fmt.Printf("Result: %d in %d ms.\n", result, took.Nanoseconds()/1e6)
```

Comment:
You could also do `took.Nanoseconds()/int64(time.Millisecond)`. Slightly more self-explanatory than the raw literal, but the cast might make it less readable.